kitsune/sumo/static/sumo/js/device-migration-wizard.js

```diff
@@ -3,6 +3,32 @@ import trackEvent from "sumo/js/analytics";
 import SwitchingDevicesWizardManager from "sumo/js/switching-devices-wizard-manager";
 import "sumo/js/form-wizard";
 
-$(document).ready(function() {
+document.addEventListener("DOMContentLoaded", function () {
   new SwitchingDevicesWizardManager(document.querySelector("#switching-devices-wizard"))
+  kbTabsInit();
 });
+
+function kbTabsInit() {
+  document.getElementById("kb-tab-all").style.display = "block";
+  let tabItems = document.querySelectorAll('[data-event-category="link click"]');
```

Comment:
Shouldn't this be just the tabs? As it's written, I think it includes all of the article links as well.

Reply:
```suggestion
  let tabItems = document.querySelectorAll('.tabs--link');
```